app/scripts/components/exploration/components/timeline/timeline.tsx

```diff
@@ -63,7 +63,7 @@ import {
   DatasetStatus,
   TimelineDatasetSuccess,
   ZoomTransformPlain
-} from '$components/exploration/types.d.ts';
+} from '$components/exploration/types.d';
```

Comment:
@sandrahoang686 do these need to be changed to `types.d.ts?`

Reply:
Looking at the file name change in this pr it appears `app/scripts/components/exploration/types.d.ts` to be accurately changes to the previous name.